### PR TITLE
#12 Kubernetes Environment

### PR DESCRIPTION
**Pull request by blaubaer**
## References
#12

### CODE REVIEW

File: pkg/alternatives/provider_default.go [hunk 1]
```diff
@@ -53,7 +33,7 @@ func (this *provider) FindBinaryFor(ctx context.Context, hostOs sys.Os, hostArch
 		if err != nil {
 			return failf(errors.System, "cannot resolve the location of the server's executable location: %w", err)
 		}
-		l.Debug("requested imp binaries does match current binary; returning itself")
+		l.Debug("requested binaries does match current binary; returning itself")
```
Comment (CiaoOtway) [inline pkg/alternatives/provider_default.go:36]: Debug("requested binaries matches current binary; returning itself")

File: pkg/alternatives/provider_default.go [hunk 2]
```diff
@@ -76,9 +56,9 @@ func (this *provider) FindBinaryFor(ctx context.Context, hostOs sys.Os, hostArch
 		if err := goos.RemoveAll(fn); err != nil {
 			return failf(errors.System, "cannot remove old alternative location file %q: %w", fn, err)
 		}
-		l.Warn("existing imp alternatives location which is broken exists and was deleted")
+		l.Warn("existing alternatives location which is broken exists and was deleted")
```
Comment (CiaoOtway) [inline pkg/alternatives/provider_default.go:59]: "existing broken alternatives' location exists and has been deleted

File: pkg/alternatives/provider_default.go [hunk 3]
```diff
@@ -87,13 +67,13 @@ func (this *provider) FindBinaryFor(ctx context.Context, hostOs sys.Os, hostArch
 		return fail(err)
 	}
 	if du == nil {
-		l.Debug("there does no existing imp alternatives exists at location and the download url resolves to empty")
+		l.Debug("there does no existing alternative at location and the download url resolves to empty")
```
Comment (CiaoOtway) [inline pkg/alternatives/provider_default.go:70]: No existing alternative location and no download url resolves to empty...
passt das so?

**Review by CiaoOtway — COMMENTED** [review]
go for it

File: cmd/bifroest/exec.go [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+	cmd.Flag("connectionId", "Connection ID this execution is connected to.").
+		Short('c').
+		PlaceHolder("<connectionId>").
+		SetValue(&opts.connectionId)
+	cmd.Flag("workingDir", "Directory to start in.").
+		Short('d').
+		Default(opts.workingDirectory).
+		PlaceHolder("<path>").
+		StringVar(&opts.workingDirectory)
+	cmd.Flag("executable", "Path to executable to be used. If not defined, first argument will be used.").
```
[the remark this answers is not in the document]
Comment (blaubaer) [reply cmd/bifroest/exec.go:41]: It means that a person should drop here the file system path which is pointing to an executable.

[the remark this answers is not in the document]
Comment (CiaoOtway) [reply cmd/bifroest/exec.go:41]: dann has to be used or is to be used

**Pull request merged**